docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :bug:`2577` Users will now be able to start a second instance of rotki without seeing an error screen.
```

Comment:
Okay so at this point I am not sure I want this. Opening rotki two times should not be encouraged. It's almost always going to be a mistake on the user side.

And if we encourage it going unnoticed we may:

(1) get DB state into an undefined state. It's not meant to be read and written by multiple instances of the same user being logged in at the same time.
(2) get their IP banned by all connected APIs since they would make `N` times the queries, not respect rate limiting etc.
(3) probably more funny stuff


So I definitely don't want to see this encouraged in the changelog. I think that opening a second instance of rotki should not be allowed. Having the port failure is a nice way to detect this so far.

Perhaps the changelog can read: 
`If the default port is taken, rotki will still be able to start`.

Which is true, and is indeed a bug this PR solves!

But then by introducing this PR we also need to find a way to detect that rotki is already running. And these 2 fixes shoudl go together.

Let me see if I can think of something.

Reply:
We can change the changelog entry, that is the easy part. then we need to create a detection mechanism to enforce a single instance of the application. I will move the PR to develop then and check if electron has anything to enforce a single instance.

Reply:
We can also try to do it on the backend @kelsos 

This is what I wanted to think. If the backend fails, so will the frontend. So the goal would be accomplished. But not sure what would the easiest cross-platform way be.

Only thing I can think of is some kind of lockfile. We already open the globalDB at some point of the app's startup. Perhaps it would make sense to write a "special setting" in the globalDB that it's used by a rotki instance and delete it when shutting down? And if the entry is there fail to start.

Then (similar to lockfiles) the problem becomes on what to do if the backend somehow stops abruptly and does not get to clean the DB entry. How do you force it? Some special argument given from the frontend? Perhaps we can discuss this in the standup tomorrow!